src/Uplink/functions.php

```diff
@@ -225,3 +227,16 @@ function get_auth_url( string $slug ): string {
 function get_license_domain(): string {
 	return get_container()->get( Data::class )->get_domain();
 }
+
+/**
+ * Get the field object for a resource.
+ *
+ * @param  Resource  $resource  The resource to get the field for.
+ *
+ * @throws \RuntimeException
+ *
+ * @return Field
+ */
+function get_field( Resource $resource ): Field {
```

Comment:
I know I told you to do this, but thinking about it again, I think we should pass a `string $slug` and grab the resource for them before setting up Field.